guides/v2.3/graphql/reference/store-config.md

```diff
@@ -193,4 +202,53 @@ The following query returns information about the store's content pages.
     }
   }
 }
+```
+
+### Query of website
+
+We can retrieve `Website` object from the [ProductInterface]({{ page.baseurl }}/graphql/reference/products.html#ProductInterface) only.
+
+**Request**
+
+```text
+{
+    products(filter: {sku: {eq: "simple"}})
```

Comment:
```suggestion
    products(filter: {sku: {eq: "24-WB04"}})
```

Reply:
I prefer to use the sample data in the examples whenever possible.

Reply:
**fixed**